app.js

```diff
@@ -13,7 +13,7 @@ const app = express();
 
 const protocol = config.node.protocol;
 const hostname = config.node.hostname;
-const port     = config.node.port;
+const port     = config.lookAndFeel.port;
```

Comment:
rename this to lookAndFeelPort 
It will be more clear

Reply:
I'm going to remove the port in its entirety as it's not need whatsoever.